testsuite/features/support/constants.rb

```diff
@@ -1326,7 +1326,7 @@
       ],
     'ubuntu-2404' =>
       %w[
-        ubuntu-24.04-pool-amd64-uyuni
+        ubuntu-2404-pool-amd64-uyuni
```

Comment:
below (line 1646) and above (line 448) it has a dot

Reply:
AFAIK this inconsistency is real and exists. Please check spacewalk-common-channels.ini .
The Community wanted it

Reply:
I'm not speaking about the inconsistency between `2204` and `24.04`.

I'm speaking about the inconsistency between `ubuntu-2404-pool-amd64-uyuni` and `ubuntu-24.04-pool-amd64-uyuni` in this file.

Reply:
So this new name for 2404 is correct @mcalmer ?

Reply:
It's the value from `spacewalk-common-channel` for the others, in the testsuite, they have no dots.

Reply:
In spacewalk-commons-channel.ini I can see 20.04, 22.04 and 24.04 has the same description.

Reply:
if all 3 have no dots in spacewalk-common-channels.ini, then current modified PR is correct. Approving.